app/helpers/__init__.py

```diff
@@ -0,0 +1,101 @@
+import time
+import boto3.exceptions as boto3_exc
+from flask import abort
+from boto3.dynamodb.conditions import Key
+from app.helpers.checks import check_and_get_url_short
+from app.helpers.response_generation import make_error_msg
+from app.models.dynamo_db import get_dynamodb_table
+from app import app
+config = app.config
+
+
+def make_api_url(scheme, host, base_path):
+    """
+    This function's goal is to ensure we can try the service in another environment than the production environment
+    It takes the request base path, the scheme and the host as a parameter
+    """
+    h = host + base_path if 'localhost' not in host else host
+    return ''.join((scheme, '://', h))
```

Comment:
This is correct. 
Since this function is now a one liner used in only one place, I'll also simply get rid of the function. (It was a function in chsdi because two services were using it)